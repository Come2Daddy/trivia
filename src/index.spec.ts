import runGoldenMaster from "jest-golden-master";

test("My first scenario", async () => {
  runGoldenMaster("my-first-scenario", async () => {
    // Write your scenario here and run the test.
    console.log("Something happens");
  });
});
