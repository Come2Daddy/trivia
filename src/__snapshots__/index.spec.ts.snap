// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`My first scenario 1`] = `
Array [
  "Mathieu was added",
  "They are player number 1",
  "Thomas was added",
  "They are player number 2",
  "Clément was added",
  "They are player number 3",
  "Mathieu is the current playerr",
  "They have rolled a 1",
  "Mathieu's new location is 1",
  "The category is Science",
  "Science Question 0",
  "Answer was correct!!!!",
  "Mathieu now has 1 Gold Coins.",
  "Thomas is the current playerr",
  "They have rolled a 1",
  "Thomas's new location is 1",
  "The category is Science",
  "Science Question 1",
  "Answer was correct!!!!",
  "Thomas now has 1 Gold Coins.",
  "Clément is the current playerr",
  "They have rolled a 1",
  "Clément's new location is 1",
  "The category is Science",
  "Science Question 2",
  "Answer was correct!!!!",
  "Clément now has 1 Gold Coins.",
  "Mathieu is the current playerr",
  "They have rolled a 1",
  "Mathieu's new location is 2",
  "The category is Sports",
  "Sports Question 0",
  "Answer was correct!!!!",
  "Mathieu now has 2 Gold Coins.",
  "Thomas is the current playerr",
  "They have rolled a 1",
  "Thomas's new location is 2",
  "The category is Sports",
  "Sports Question 1",
  "Answer was correct!!!!",
  "Thomas now has 2 Gold Coins.",
  "Clément is the current playerr",
  "They have rolled a 1",
  "Clément's new location is 2",
  "The category is Sports",
  "Sports Question 2",
  "Answer was correct!!!!",
  "Clément now has 2 Gold Coins.",
  "Mathieu is the current playerr",
  "They have rolled a 1",
  "Mathieu's new location is 3",
  "The category is Rock",
  "Rock Question 0",
  "Answer was correct!!!!",
  "Mathieu now has 3 Gold Coins.",
  "Thomas is the current playerr",
  "They have rolled a 1",
  "Thomas's new location is 3",
  "The category is Rock",
  "Rock Question 1",
  "Answer was correct!!!!",
  "Thomas now has 3 Gold Coins.",
]
`;
